docs/osf-trivia-list/dts.md

```diff
@@ -91,3 +91,19 @@ Please [donate](../ways-you-can-help-us.md#donate-money) using one of
 available methods and contact [3mdeb](mailto:leads@3mdeb.com) or [Dasharo
 Team](mailto:contact@dasharo.com) to let us know about your donation. Minimal
 donating to access DTS SE with one year update support is 60EUR.
+
+## How can I help support diagnose an issue I'm having quicker?
```

Comment:
That sentence should probably be redone because currently it sounds really weird (at least to me). Maybe something like `How can I help with diagnosis to make it go faster?`